src/main/java/core/basesyntax/shape/RightTriangle.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax.shape;
+
+import core.basesyntax.type.Color;
+
+public class RightTriangle extends Shape {
+    private double base1;
+    private double base2;
+
+    public RightTriangle(Color color, double base1, double base2) {
```

Comment:
Update the constructor parameter names to 'leg1' and 'leg2' to match the class variable names.